### ISSUE TITLE
vim.api.nvim_set_hl invalid key: inverse

### ISSUE THREAD

**Issue by mehalter** [EdenEast/nightfox.nvim#136]
## Description

I pulled the latest neovim 0.7 support and got an error in my compilation. It seems that the style `inverse` has been renamed to `reverse` 

## Behavior

**Expected behavior**
```
Error detected while processing /home/micah/.config/nvim/init.lua:
E5113: Error while calling lua chunk: /home/micah/.config/nvim/init.lua:25: Failed to load core.options

/home/micah/.config/nvim/lua/core/options.lua:9: Vim(lua):E5108: Error executing lua ...te/pack/packer/start/nightfox.nvim/lua/nig
htfox/main.lua:58: Vim(luafile):E5113: Error while calling lua chunk: /home/micah/.cache/nvim/nightfox/duskfox_compiled.lua:146: i
nvalid key: inverse
stack traceback:
        [C]: in function 'nvim_set_hl'
        /home/micah/.cache/nvim/nightfox/duskfox_compiled.lua:146: in main chunk
        [C]: in function 'cmd'
        ...te/pack/packer/start/nightfox.nvim/lua/nightfox/main.lua:58: in function 'load'
        [string ":lua"]:18: in main chunk
        [C]: in function 'nvim_command'
        /home/micah/.config/nvim/lua/core/options.lua:9: in main chunk
        [C]: at 0x7f13bb47f9d0
        [C]: in function 'pcall'
        /home/micah/.config/nvim/init.lua:23: in main chunk
stack traceback:
        [C]: in function 'cmd'
        ...te/pack/packer/start/nightfox.nvim/lua/nightfox/main.lua:58: in function 'load'
        [string ":lua"]:18: in main chunk
        [C]: in function 'nvim_command'
        /home/micah/.config/nvim/lua/core/options.lua:9: in main chunk
        [C]: at 0x7f13bb47f9d0
        [C]: in function 'pcall'
        /home/micah/.config/nvim/init.lua:23: in main chunk
stack traceback:
        [C]: in function 'error'
        /home/micah/.config/nvim/init.lua:25: in main chunk
```

## Configuration

```lua
return {
  options = {
    inverse = {
      match_paren = true,
    },
  },
}
```

**Reproduce**

1. Set up `NightFox`  with an  `inverse` option set
1. run `:NightfoxCompile`
1. restart neovim 0.7

## Additional info / context

When I go to my compiled file and replace `inverse = true` with `reverse = true` it works as intended. I got the  hint from the docs here: https://neovim.io/doc/user/syntax.html#inverse

**Comment by EdenEast** [comment]
Just merged PR #137 that fixes this issue. Since the `highlight` command can handle both I just switched it to use reverse instead of inverse.

**Comment by mehalter** [comment]
Thanks!
